.github/workflows/irs-build.yml

```diff
@@ -156,11 +156,13 @@ jobs:
           IMAGE_NAME: irs-api
         if: env.DOCKER_HUB_USER != '' && github.event_name != 'pull_request'
         run: |
-          docker tag ${{ matrix.image }} ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ steps.version.outputs.image_tag }}
-          docker push ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ steps.version.outputs.image_tag }}
+          IMAGE_ID=${{ needs.init.outputs.image_namespace }}/${{ needs.init.outputs.image_name }}
```

Comment:
where does `init.outputs.image_namespace` come from? Image name and namespace are defined in the env: section of this step